touchforms/backend/touchcare.py

```diff
@@ -390,6 +160,33 @@ def filter_cases(filter_expr, api_auth, session_data=None, form_context=None):
         return {'cases': filter(lambda x: x, case_list.split(","))}
     except (XPathException, XPathSyntaxException), e:
         raise TouchcareInvalidXPath('Error querying cases with xpath %s: %s' % (filter_expr, str(e)))
+    except Exception, e:
+        print "E: ", e
```

Comment:
that was just leftover from debugging, removed (unless you think we actually should catch... erroring out seems right though)